kernel/kernel-pdfgenerator/pom.xml

```diff
@@ -64,7 +64,46 @@
 			<groupId>com.itextpdf.tool</groupId>
 			<artifactId>xmlworker</artifactId>
 			<version>${itext.version}</version>
+		</dependency>-->
```

Comment:
Let us remove the itext dependency block as it's not going to be used further.

Reply:
taken care